openshift/route-monitor-operator
# feat(routemonitor): add deletion capabilities

- also copied from aws-account-operator the pkg/util/finalizer/finalizer.go
- made tests pass but not added tests for new code

TO TEST:
1.  run `make` and `make test` to test build and deploy
2. change Makefile and config/manager/manager.yaml to point to the repo you wish to use
3.  build and deploy with `make docker-build` and `make docker-push`
4. use `make deploy` to deploy your operator on a cluster you are logged into
5. check logs with `oc logs deploy/route-monitor-operator-controller-manager -c manager`
